src/main/java/net/atos/zac/app/informatieobjecten/model/RESTInformatieObjectZoekParameters.java

```diff
@@ -17,4 +17,6 @@ public class RESTInformatieObjectZoekParameters {
     public UUID[] UUIDs;
 
     public boolean toonGekoppeldeZaakDocumenten;
+
+    public boolean searchGekoppeldeZaak;
```

Comment:
Ik vind het mooi dat je er nu voor hebt gezorgd dat dat ophalen van die zaak alleen gebeurd wanneer het nodig is. Maar de naam die je gekozen hebt voor deze boolean geeft mij heel weinig clue waar ie voor is.

Er zijn twee mogelijkheden:
- Hij geeft aan in welke context de aanroep gebeurt, en dan zou ie een naam moeten hebben die dat helemaal duidelijk maakt.
- Hij geeft aan wat er gebeurt als ie true is, en ook dan zou ie een naam moeten hebben waaruit je dat kunt aflezen.

Reply:
Opgelost in nieuwe commit